install.sh

```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
 echo "# to test the installation (assuming your shell is bash):               #"
 echo "# $ source $conda_prefix/bin/activate dgfem                             #"
 echo "# $ export OCL_ICD_VENDORS=$MY_CONDA_DIR/envs/dgfem/etc/OpenCL/vendors  #"
-echo "# $ python mirgecom/examples/wave-eager.py                              #"
+echo "# $ python /path/to/mirgecom/examples/wave-eager.py                     #"
```

Comment:
This didn't end up cloning/installing mirgecom (AFAICT). I think it should.

Reply:
Yes, this currently doesn't clone mirgecom. I can add that before cloning the other repos; in this case, we wouldn't even need to `wget` the requirements.txt (but just use the one from the mirgecom clone).

Reply:
I'd say go for it.

Reply:
Done.